bundles/http/Citrix/9ec6b7b5-8aad-4b5a-929b-d6a59cf8e7ad/metadata.json

```diff
@@ -2,53 +2,53 @@
   "type" : "HTTP",
   "vendor" : "Citrix",
   "id" : "9ec6b7b5-8aad-4b5a-929b-d6a59cf8e7ad",
-  "title" : "ServiceNow HTTP",
+  "title" : "ServiceNow",
```

Comment:
Toto je ok?

Reply:
This change was initiated by and agreed with Chandrika Srinivasan on 2020/06/11